.replit

```diff
@@ -0,0 +1,14 @@
+run = ["just", "test"]
+
+modules = ["go-1.21:v2-20231201-3b22c78"]
```

Comment:
This repo requires Go 1.22

Reply:
Let me see if we can get an updated Go module out for 1.22

Reply:
I see you're adding Hermit to Nix, presumably to enable its use through Replit? That would definitely be our ideal path going forward as we're managing all of our toolchain versions through Hermit. But this works for now.

Reply:
Yeah that's the ideal plan here, hopefully can get it into nixpkgs and available on Replit here soon

Reply:
Thanks for doing that!